h2o-core/src/main/java/hex/grid/hyperspace/RandomDiscreteValueWalker.java

```diff
@@ -0,0 +1,185 @@
+package hex.grid.hyperspace;
```

Comment:
same as for CartesianWalker, if any change from original, please show us.

Reply:
Replied https://github.com/h2oai/h2o-3/pull/3990#discussion_r340614774